gcp-observability/src/main/java/io/grpc/gcp/observability/interceptors/InternalLoggingChannelInterceptor.java

```diff
@@ -40,7 +40,7 @@
 import java.util.logging.Logger;
 
 /**
- * A logging interceptor for {@code LoggingChannelProvider}.
+ * A logging client interceptor for Observability.
```

Comment:
As mentioned elsewhere we should consider getting rid of the Factory (in the Server interceptor as well). A separate PR seems more convenient?

Reply:
I do agree and will do this in another PR.

Reply:
Pls add a TODO comment saying so

Reply:
Done